lib/src/widgets/date_range_picker.dart

```diff
@@ -139,7 +140,12 @@ class DateRangePickerWidget extends StatefulWidget {
     this.height = 329,
     this.displayMonthsSeparator = true,
     this.separatorThickness = 1,
-    this.allowSingleTapDaySelection = false,
+    this.firstDayOfWeek = 0,
+  })  : assert(
+          firstDayOfWeek >= 0 && firstDayOfWeek <= 6,
+          'firstDayOfWeek must be in the range [0..6].',
+        ),
+        this.allowSingleTapDaySelection = false,
```

Comment:
Here is in Error which should be fixed soon. Otherwise it isn't useable. 

Kind regards

Reply:
what error?

Reply:
@marcel-ploch-coodoo could you please clarify what is the error?

Reply:
I am sorry. I had an error here but it seems to be working now

Reply:
no worries, thanks for reviewing :smile: